tiproxy/tiproxy-command-line-flags.md

```diff
@@ -195,17 +198,26 @@ tiproxyctl traffic replay --host 10.0.1.10 --port 3080 --username="u1" --passwor
 
 #### `traffic cancel`
 
-The `tiproxyctl traffic cancel` command is used to cancel the current capture or replay task.
+The `tiproxyctl traffic cancel` command is used to cancel the current capture or replay job.
 
 #### `traffic show`
 
-The `tiproxyctl traffic show` command is used to display historical capture and replay tasks.
-
-The `status` field in the output indicates the task status, with the following possible values:
-
-- `done`: the task completed normally.
-- `canceled`: the task was canceled. You can check the `error` field for the reason.
-- `running`: the task is running. You can check the `progress` field for the completion percentage.
+The `tiproxyctl traffic show` command is used to display historical capture and replay jobs. It outputs an array of objects, and each object represents a job. Each job has the following fields:
+
+- `type`: the job type. `capture` indicates a traffic capture job, `replay` indicates a traffic replay job
+- `start_time`: the start time of the job
+- `end_time`: the end time if the job has finished, otherwise it is empty
+- `duration`: the duration of the traffic capture job
+- `output`: the output traffic file path of the capture job
+- `encryption_method`: the encryption method of the traffic file
+- `compress`: whether the traffic files are compressed
+- `input`: the input traffic file path of the replay job
+- `username`: the database username for traffic replay
+- `speed`: the replay speed multiplier
+- `read_only`: whether only replays read-only statements
+- `progress`: the completion percentage of the job
+- `status`: the current status of the job. `running` means it is running, `done` means it is completed normally, and `canceled` means the job failed
+- `error`: if the job fails, this column contains the reason for the failure, otherwise it is empty. For example, `manually stopped` means the user manually canceled the job by executing `CANCEL TRAFFIC JOBS`
```

Comment:
```suggestion
- `error`: if the job fails, this column contains the reason for the failure. Otherwise it is empty. For example, `manually stopped` means the user manually cancels the job by executing `CANCEL TRAFFIC JOBS`.
```